rero_ils/modules/ebooks/dojson/contrib/marc21/model.py

```diff
@@ -233,7 +233,8 @@ def marc21_to_title(self, key, value):
                     if subtitle:
                         title_data['subtitle'] = subtitle
                 elif not subfield_246_a:
-                    title_data['subtitle'] = value_data
+                    if value_data:
```

Comment:
Maybe you could merge two 'if' conditions into one line ?

Reply:
ok